addons/GearSwap/refresh.lua

```diff
@@ -151,16 +151,16 @@ function load_user_files(job_id,user_file)
     user_env['_G'] = user_env
     
     -- Try to load data/<name>_<main job>.lua
-    local funct, err = loadfile(path)
+    local funct, err = path and loadfile(path) or function() end
     
     -- If the file cannot be loaded, print the error and load the default.
-    if funct == nil then
+    if err then
         print('User file problem: '..err)
         current_file = nil
-        gearswap_disabled = true
+        -- gearswap_disabled = true
         sets = nil
         return
-    else
+    elseif filename then
```

Comment:
Since the `else` is irrelevant here you can change this into an
```lua
end

if filename then
```

Reply:
I took it out entirely, turns out it's not needed at all!